import "combat.proto";

package tutorial.orcas.combat.server.vote;

// VOTE_COMBAT_BEGIN
message VoteCombatBegin {
  required int32 combat_id = 1;                                   // 战斗id
}

// VOTE_COMBAT_BUILD_ACTION
message VoteCombatBuildAction {
  required int32 combat_id = 1;                                   // 战斗id
  required int32 building_id = 2;                                 // 建筑id
  required int32 warrior_conf_id = 3;                             // 士兵配置id
}

// VOTE_COMBAT_ATTACK_ACTION
message VoteCombatAttackAction {
  required int32 combat_id = 1;                                   // 战斗id
  required int32 warrior_id = 2;                                  // 士兵id
  required .protocol.CombatTarget target = 3;                     // 目标
}
